docs/CONTRIBUTING.md

```diff
@@ -43,12 +43,6 @@ Every PR should be reviewed and approved by someone other than the author, even
 
 The recommended way of finding an appropriate person to review your code is by [blaming](https://help.github.com/articles/using-git-blame-to-trace-changes-in-a-file/) one of the files you are updating and looking at who was responsible for previous commits on that file.
 
-Then, you may ask that person to review your code by mentioning their GitHub username on the PR comments like this:
```

Comment:
I'm guessing this predates the possibility to assign a reviewer in GitHub

Reply:
Interesting. It's worth noting that folks without write permissions sometimes don't have permissions to request a review from anyone. Could be something to keep in mind.